samples/excel/85-preview-apis/pivottable-pivotfilters.yaml

```diff
@@ -0,0 +1,326 @@
+              ["D Farms", "Apple", "Conventional", "7/23/2020", 100, 3000],
+              ["D Farms", "Apple", "Organic", "7/26/2020", 80, 2800],
+              ["E Farms", "Lime", "Conventional", "8/1/2020", 160, 2700],
+              ["E Farms", "Orange", "Conventional", "8/7/2020", 180, 2000],
+              ["E Farms", "Apple", "Conventional", "8/10/2020", 245, 2200],
+              ["E Farms", "Kiwi", "Conventional", "8/11/2020", 200, 1500],
+              ["F Farms", "Kiwi", "Organic", "7/30/2020", 100, 150],
+              ["F Farms", "Lemon", "Conventional", "7/14/2020", 150, 270]
+            ];
+
+            const range = dataSheet.getRange("A1:F21");
+            range.values = data;
+            range.format.autofitColumns();
+            pivotSheet.activate();
+
+            // Create the PivotTable.
+            context.workbook.worksheets.getActiveWorksheet().pivotTables.add("Farm Sales", "Data!A1:F21", "A2");
+            const pivotTable = context.workbook.worksheets.getActiveWorksheet().pivotTables.getItem("Farm Sales");
+            pivotTable.rowHierarchies.add(pivotTable.hierarchies.getItem("Farm"));
+            pivotTable.rowHierarchies.add(pivotTable.hierarchies.getItem("Type"));
+            pivotTable.dataHierarchies.add(pivotTable.hierarchies.getItem("Crates Sold at Farm"));
+            pivotTable.dataHierarchies.add(pivotTable.hierarchies.getItem("Crates Sold Wholesale"));
+
+            await context.sync();
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to filter PivotTables with the different PivotFilters.</p>
+        </section>
+
+        <section class="setup ms-font-m">
+          <h3>Set up</h3>
+          <button id="setup" class="ms-Button">
+                <span class="ms-Button-label">Setup sample</span>
+            </button>
+        </section>
+
+        <section class="setup ms-font-m">
+          <h3>Try it out</h3>
+        </section>
+
+        <section class="setup ms-font-m">
+          <p><b>Filters</b></p>
+          <button id="manual-filter" class="ms-Button">
+              <span class="ms-Button-label">Organic filter (Manual)</span>
+          </button>
+          <p />
+          <button id="date-filter" class="ms-Button">
+              <span class="ms-Button-label">Before August filter (Date)</span>
+          </button>
+          <p />
+          <button id="value-filter" class="ms-Button">
+              <span class="ms-Button-label">>500 units wholesale filter (Value)</span>
```

Comment:
The button labels are a little confusing to me. I think it's because two of them are exclusive (removing events before August and removing lemons + limes) while the other is inclusive (only showing rows with more than 500 wholesale crates). At first I thought the sample wasn't working correctly, but after investigation I think tweaking the button labels would solve my confusion. Maybe something like: "Filter out events before August", "Filter out rows with less than 500 units wholesale", and "Filter out lemons/limes"